Singularity/Singularity/Platforms/PlatformFactory.cs

```diff
@@ -74,7 +76,7 @@ public static void Init(Texture2D coneSheet, Texture2D cylinderSheet, Texture2D
                     return new PlatformBlank(position, sBlankSheet, sBlankSheet, sLibSans12, ref director, friendly: friendly);
 
                 case EPlatformType.Command:
-                    return new CommandCenter(position, sCylinderSheet, sBlankSheet, sLibSans12, ref director, friendly);
+                    return new CommandCenter(position, sCylinderSheet, sBlankSheet, sLibSans12, ref director, commandBlueprint);
```

Comment:
just wondering why you took out the friendly specification here?

Reply:
I thought the idea was that the Enemy (structures) only has Defending (Laser or Kinetic) as well as Spawning (Barracks) Platforms. Thus, In our case there'd be no actual case where the Enemy would have Command centers (or other in this case specific structures for us). Still, yes, maybe it would have been better to leave it the more general way.